Mathlib/Data/Multiset/Sort.lean

```diff
@@ -71,6 +71,9 @@ theorem sort_cons (a : α) (s : Multiset α) :
   refine Quot.inductionOn s fun l => ?_
   simpa [mergeSort_eq_insertionSort] using insertionSort_cons r (a := a) (l := l)
 
+theorem sort_range (n : ℕ) : sort (· ≤ ·) (range n) = List.range n :=
```

Comment:
This should be `simp`

Reply:
This doesn't work, as `simp` eagerly simplifies the LHS to `mergeSort (List.range n) = List.range n`. That feels like it shouldn't happen.

Reply:
Actually, the issue might be the LHS is `List.range` rather than `Multiset.range`. I'll take a look.

Reply:
This was indeed the issue, and has been fixed.